src/test/java/uk/gov/hmcts/reform/adoption/adoptioncase/schedule/AlertDraftCaseApplicantBeforeDeletionTaskTest.java

```diff
@@ -89,11 +88,6 @@ void run() {
         final CaseDetails caseDetails1 = mock(CaseDetails.class);
         final CaseDetails caseDetails2 = mock(CaseDetails.class);
         final CaseDetails caseDetails3 = mock(CaseDetails.class);
-
-        when(caseDetails1.getCreatedDate()).thenReturn(LocalDateTime.now().minusDays(83));
-        when(caseDetails2.getCreatedDate()).thenReturn(LocalDateTime.now().minusDays(83));
-        when(caseDetails3.getCreatedDate()).thenReturn(LocalDateTime.now().minusDays(83));
-
```

Comment:
PT, follow up to check that we don't need these.

Reply:
This is correct.  The query to retrieve cases is mocked, so that the date is not considered.
Including the date give a **org.mockito.exceptions.misusing.UnnecessaryStubbingException**